<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>964</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="RankingTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>1091</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranking</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Database_select_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select dataset</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Preprocessing_select_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>select preprocessing</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Algorithm_select_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>select algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_path_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>36</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="load_model_button">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>670</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_filename_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>No File</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="dataset_scrollarea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>719</width>
       <height>159</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="x_test_scrollarea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>719</width>
       <height>159</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="x_test_scrollarea_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>49</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected x_test</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="accuracy_f1score">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>accuracy / f1_score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y_test</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="y_test_textBrowser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="train_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start train</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hidden_layer_select">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>relu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sigmoid</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="output_layer_select">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>sigmoid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>softmax</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>linear</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x_test</string>
    </property>
   </widget>
   <widget class="QLabel" name="selected_preprocessing_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>hidden layer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>output layer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress list</string>
    </property>
   </widget>
   <widget class="QWidget" name="loss_accuracy" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>511</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="selected_algorithm_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_ranking_button">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>860</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Ranking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_ranking_button">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>820</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Ranking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_model_button">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>630</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Model</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="actiontest_menu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actiontest_menu">
   <property name="text">
    <string>test menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
